### ISSUE TITLE
DWDWeather is overriding deprecated methods on an instance of WeatherEntity, this is not valid and will be unsupported from Home Assistant 2023.1

### ISSUE THREAD

**Issue by FaserF** [FL550/dwd_weather#58]
## Version of home_assistant

HA 2022.07.0 Running on the latest HA OS 

## Version of the custom_component

1.2.22

## Describe the bug
A clear and concise description of what the bug or error is and when it happens.

Log warning about deprecating feature. This „bug“ is low priority and more a hint from my side from the logs, coming with the newest HA release.

## Debug log

```text

Logger: homeassistant.components.weather
Source: components/weather/__init__.py:308 
Integration: Weather (documentation, issues) 
First occurred: 22:04:44 (1 occurrences) 
Last logged: 22:04:44

custom_components.dwd_weather.weather::DWDWeather is overriding deprecated methods on an instance of WeatherEntity, this is not valid and will be unsupported from Home Assistant 2023.1. Please report it to the custom component author.

```

**Comment by FL550** [comment]
Is anyone still having this message? I am unable to reproduce the error message.

**Comment by DeepCoreSystem** [comment]
Yes, I am having this message each time I restart Home Assistant.
It seems that it does get logged only once when HA reloads the component.
I did right now a restart, here is a copy of the log entry:

```
Logger: homeassistant.components.weather
Source: components/weather/__init__.py:198
Integration: Weather (documentation, issues)
First occurred: 13:25:14 (1 occurrences)
Last logged: 13:25:14

custom_components.dwd_weather.weather::DWDWeather is overriding deprecated methods on an instance of WeatherEntity, this is not valid and will be unsupported from Home Assistant 2023.1. Please report it to the custom integration author.
```

This is the actual setup I use:

```
DWD V1.2.22
Home Assistant 2022.8.7 
Supervisor 2022.08.5
Operating System 8.5
Frontend 20220802.0 - latest 
```

**Comment by FL550** [comment]
Thanks for your quick answer. I'll fix this today

**Comment by FL550** [comment]
Update is out, this should resolve the message.
